### DISCUSSION TITLE
re-run output postprocessing

### DISCUSSION THREAD

**Discussion by mikapfl** [magpiemodel/magpie#384] [Q&A] [answered]
Hi,

I would like to re-run the output postprocessing without re-running the whole model. In particular, I want to re-generate the `.mif` files. I can't really figure out how to do it. Should I just run everything listed in `config/default.cfg` in `cfg$output` via `output.R`? Or which of these is the relevant one?
```R
cfg$output <- c("output_check", "rds_report", "validation_short",
                "extra/disaggregation")
```

Thanks a lot + Cheers,

Mika

**Comment by abhimishr** [comment, marked as answer]
Hi Mika,

I think just running the rds_report should do the trick for report.mif at least. It should make both report.rds and report.mif files but I noticed that report.rds files are not overwritten if they already exist so might be worthwhile to delete/rename the existing report.rds and report.mif files. validation.mif is copied over from the validation tgz file I think. You would need another validation tgz in case you want to update the validation.mif and extract it from there.

**Reply by mikapfl** [reply]
I ended up running rds_report, but had to comment out some code which submitted the run to the central run database, so I wouldn't submit the same run mutliple times.

**Comment by flohump** [comment]
Hi Mika,

for re-generating the mif/csv file, you only need “ rds_report”

Best,
Florian
